tests/testdata/views/deployments.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: mission-control.flanksource.com/v1
+kind: View
+metadata:
+  name: deployments
+  namespace: mc
+  uid: 6d53ff5a-13ec-4049-8a97-adad0fcd97ca
+spec:
+  columns:
+    - name: application
+      type: string
+      description: The application name.
+      primaryKey: true
+    - name: namespace
+      type: string
+      description: The namespace name.
+    - name: chart
+      type: string
+      description: The chart name.
+    - name: version
+      type: string
+      description: The version of the deployment.
+    - name: status
+      type: status
+      description: The status of the deployment.
+    - name: health
+      type: health
+      description: The health of the deployment.
+    - name: lastUpdated
+      type: datetime
+      description: The last updated time.
+  queries:
+    configs:
+      - selector:
+          search: '@order=name'
+          types:
+            - Kubernetes::HelmRelease
+          tagSelector: namespace in (monitoring,media)
+        mapping:
+          application: row.name
+          namespace: row.tags.namespace
+          chart: row.config.status.history[0].chartName
```

I think this would be nullsafe
```suggestion
          chart: jsonpath(row.config, '$.status.history[0].chartName')
```

---

`Select *,  configs.config->status.history[0].chartName`